app/javascript/pages/Home.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import getAllHotels from "../Redux/HomePageAction";
+import { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import NavBar from "../components/Navbar";
+
+const Home = () => {
```

Comment:
- [ ] On mobile, your homepage has considerable overflow on some elements:


![image](https://user-images.githubusercontent.com/57421823/201336607-dce5f152-3b81-4f8a-8e91-831fa513a11b.png)

![image](https://user-images.githubusercontent.com/57421823/201336831-b23218b6-375f-420b-a398-1787d699aa4b.png)

There seems to be an issue with the carrousel because when I swipe it, the whole page scrolls horizontally.

Kindly fix this issue so your mobile design is more usable and best looking 👍